pkg/graphql/graphql.go

```diff
@@ -0,0 +1,68 @@
+// Package graphql for grid graphql support
+package graphql
```

Comment:
Did we really needed to implement this from scratch ? can't we use an existing graphql client ?